client/js/navigation/config.js

```diff
@@ -23,7 +26,30 @@ export const sharedNavigationOptions = navigation => ({
   },
   headerLeft: (
     <TouchableOpacity onPress={() => navigation.toggleDrawer()}>
-      <Icon name="md-menu" size={25} color="white" style={{ margin: 15 }} />
+      <IconEntypo
+        name="menu"
+        size={45}
+        color="#fff"
+        style={{ marginLeft: 10 }}
+      />
+    </TouchableOpacity>
+  ),
+  headerTitle: (
+    <TouchableOpacity>
+      <Image
+        source={require("../assets/images/bikelockerlogo-white.png")}
+        style={{ width: 40, height: 40, marginBottom: 5 }}
```

Comment:
Move to a style object

Reply:
This was part of the navigation feature.